src/components/VideoProvider/index.tsx

```diff
@@ -81,7 +81,8 @@ export function VideoProvider({ options, children, onError = () => {} }: VideoPr
   useRestartAudioTrackOnDeviceChange(localTracks);
 
   const [isBackgroundSelectionOpen, setIsBackgroundSelectionOpen] = useState(false);
-  const [backgroundSettings, setBackgroundSettings] = useBackgroundSettings();
+  const videoTracks = localTracks.find(track => track.name.includes('camera')) as LocalVideoTrack;
```

Comment:
Nice idea to grab the camera track from here! One small thing: it's possible that `localTracks.find` returns `undefined`, so I think the type of this variable needs to be adjusted. 

`const videoTracks = localTracks.find(track => track.name.includes('camera')) as LocalVideoTrack | undefined;`